apps/tx-builder/src/pages/Dashboard.tsx

```diff
@@ -0,0 +1,227 @@
+import { ReactElement, useState, useEffect } from 'react';
+import { Text, Title, Link, AddressInput, Button } from '@gnosis.pm/safe-react-components';
+import styled from 'styled-components';
+import InputAdornment from '@material-ui/core/InputAdornment';
+import Grid from '@material-ui/core/Grid';
+import CheckCircle from '@material-ui/icons/CheckCircle';
+
+import { ContractInterface } from '../hooks/useServices/interfaceRepository';
+import useServices from '../hooks/useServices';
+import { isValidAddress } from '../utils';
+import AddNewTransactionForm from '../components/forms/AddNewTransactionForm';
+import TransactionsBatchList from '../components/TransactionsBatchList';
+import CreateNewBatchCard from '../components/CreateNewBatchCard';
+import JsonField from '../components/forms/fields/JsonField';
+import { ProposedTransaction } from '../typings/models';
+
+type DashboardProps = {
+  transactions: ProposedTransaction[];
+  handleSubmitTransactions: () => void;
+  handleAddTransaction: (transaction: ProposedTransaction) => void;
+  handleRemoveTransaction: (index: number) => void;
+  handleRemoveAllTransactions: () => void;
+  handleReorderTransactions: (sourceIndex: number, destinationIndex: number) => void;
+};
+
+const Dashboard = ({
```

Comment:
It is the same old `Dashboard` component, I just move it to the new `pages` folder